npm/ux/circuit.tsx

```diff
@@ -18,3 +18,27 @@ export function Circuit(props: { circuit: qviz.Circuit }) {
 
   return <div class="qs-circuit" ref={circuitDiv}></div>;
 }
+
+export function CircuitPanel(props: {
+  title: string;
+  subtitle: string;
+  circuit?: qviz.Circuit;
+  errorHtml?: string;
+}) {
+  return (
+    <div>
+      <div>
+        <h1>{props.title}</h1>
+        <h2>{props.subtitle}</h2>
+      </div>
+      {props.circuit ? <Circuit circuit={props.circuit}></Circuit> : <></>}
```

Comment:
Typically you just return `null` (rather than `<></>`) if you don't want content in the alternate case.

Reply:
Thanks, I didn't know this was possible.